site/src/pages/SearchPage/SearchPage.scss

```diff
@@ -12,11 +12,26 @@
 
     #search-list {
         width: 50vw;
+        display: flex;
+        flex-direction: column;
+        height: 85vh;
+        overflow-y: hidden;
     }
 
     #search-popup {
         flex-grow: 1;
         height: fit-content;
+        position: sticky;
+        top: 0;
```

Comment:
Now that both the search list and popup have scrollbars within the containers, the popup itself doesn’t move anymore! Because of that, we no longer need to ensure the popup stops at the very top when scrolling down on the page (which is what these two lines accomplish).

I played around with the webpage after disabling these two properties in the web inspector, and the behavior’s exactly the same without them.
```suggestion
```